src/components/messageListener.ts

```diff
@@ -0,0 +1,8 @@
+import { Message } from 'discord.js';
+import { applyBonusByUserId } from './coin';
+
+export const messageListener = async (message: Message): Promise<void> => {
+  if (message.author.id != '843696481826373633') {
```

Comment:
hard coded ID?

Reply:
we may also want to make sure the author is not a bot

Reply:
wups I forgot to change this
added to .env
also changed readme

Reply:
In addition, I think it would be good if only guild messages lead to a user gaining Codey coins, since currently, a user can also gain Codey coins via DMs with the bot.

Reply:
I'm pretty sure you can do `client.user.id` to get the ID. Let's try to avoid env vars as much as possible because it involves configuration values for each deployment environment.

Reply:
https://discord.js.org/#/docs/main/v12/class/Client?scrollTo=user